pages/common/ssh.md

```diff
@@ -1,7 +1,7 @@
 # ssh
 
 > Secure Shell is a protocol used to securely log onto remote systems.
-> It can be used for logging or executing commands on a remote server.
+> This command also has documentation about its subcommands, e.g. `ssh-add`.
```

Comment:
I check that all the `ssh-*` commands in tldr are not subcommands from `ssh`.

Reply:
But can we consider them to be sub-command-ish enough to be mentioned?

Reply:
Hrm, I wouldn't say so. They are completely separate programs, and this is also a prime example that e.g. on the `git` page as to why we should have "e.g. `tldr git merge`" instead of "e.g. `tldr git-merge`".

What about this?
```suggestion
> See also related commands, such as `ssh-add` and `ssh-keygen`.
```

Reply:
Sounds good, but I think this should be done in a separate PR. I'll remove ssh from the list.